control_panel_api/services.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+
+from . import aws
+
+READ_WRITE = 'readwrite'
+READ_ONLY = 'readonly'
+
+
+def _bucket_name(name):
+    """Prefix the bucket name with environment e.g. dev-james"""
+    return "{}-{}".format(settings.ENV, name)
+
+
+def _policy_name(name, readwrite=False):
+    """Prefix the policy name with bucket name, postfix with access level e.g. dev-james-readwrite"""
+    return "{}-{}".format(_bucket_name(name), READ_WRITE if readwrite else READ_ONLY)
+
+
+def create_team_bucket(name):
```

Comment:
This method name should be more generic now that we can also create S3 buckets for apps (or for any reason really). Maybe just call is `create_bucket`, what do you reckon?

Reply:
👍